### ISSUE TITLE
Can't update className prop

### ISSUE THREAD

**Issue by AndyOGo** [sarneeh/reaptcha#189]
I have a form, which should disable all fields during submission.
But I can't update `className` prop.

```js
render() {
    const {
      meta: { submitting },
    } = this.props;

    return (
      <Reaptcha
        className={classnames('m-aletheia-captcha', {
          'm-aletheia-captcha--disabled': submitting,
        })}
     />
  }
```

**Comment by sarneeh** [comment]
But what specifically does not work? The class is not there in HTML?

**Comment by AndyOGo** [comment]
Yes, the `className` DOM property or HTML class attribute is not updated.

**Comment by sarneeh** [comment]
But where are you looking for the class? Because it is applied to the container where recaptcha sits, not the recaptcha itself. You can't apply a class to the recaptcha component.

**Comment by AndyOGo** [comment]
I'm looking for it on the container.

If I understood the docs correctly, I assumed that I can dynamically change the `className`.
> Classname for the container element
https://github.com/sarneeh/reaptcha#customisation

**Comment by sarneeh** [comment]
@AndyOGo It works without a problem on my side.

![image](https://user-images.githubusercontent.com/11553624/70551077-22600e00-1b77-11ea-934a-e1d4e30f2343.png)

**Comment by AndyOGo** [comment]
@sarneeh 
Yeah a static `className="static-class"` works for me too.
Though if I want to assign a dynamically gerated `className={fooOrBar ? 'foo' : 'bar'}`, that's is not working upon updates.

I will take the time to provide a reproduction.

**Comment by AndyOGo** [comment]
@sarneeh 
Here you go:
https://codesandbox.io/s/practical-taussig-7jn7f?fontsize=14&hidenavigation=1&theme=dark

**Comment by sarneeh** [comment]
@AndyOGo Ohh... I'm sorry! I know where the problem is.

**Comment by sarneeh** [comment]
Released in `1.7.2`.
